HW_7/Task 1.js

```diff
@@ -0,0 +1,90 @@
+// Task 1
+function mergedArray(...arrays) {
+    let result = [];
+    for (const array of arrays) {
+        for (const element of array) {
+            result.push(element);
+        }
+    }
+    return result;
+}
+
+const r = mergedArray([1, 2], [3, 4], [5, 6]);
+
+console.log(r);
+
+
+function mergedArray(...arrays) {
+    return [].concat(...arrays);
+}
+console.log(mergedArray([1, 2], [3, 4], [5, 6]));
+
+// Task 2
+// 1 solution
+const sentence = "I am super engineer";
+function sentenceProcess(sentence) {
+    let result = "";
+    for (let i = 0; i < sentence.length; i++) {
+        const symbol = sentence[i];
+        if (i === 0) {
+            result = result.concat(symbol.toLowerCase());
+        } else if (sentence[i - 1] === " ") {
+            result = result.concat(symbol.toUpperCase());
+        } else if (symbol === " ") {
+            result = result.concat("_");
+        } else {
+            result = result.concat(symbol);
+        }
+    }
+
+    return result;
+}
+console.log(sentenceProcess(sentence));
+
+// 2 solution
+function upperCaseFirst(str) {
+    if (!str) return str;
+
+    return str[0].toUpperCase() + str.slice(1);
```

Comment:
А если попадется такое предложение `I aM suPer EnginEeR"`, то можно немного поправить
```suggestion
    return str[0].toUpperCase() + str.slice(1).toLowerCase();
```